src/components/Copyable.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropsWithChildren, useRef } from 'react'
+import { copyToClipboard } from '../utils'
+
+export interface CopyableProps {
+  value: string
+  onSuccess?: () => void
+  onError?: (e: Error) => void
+  className?: string
+}
+
+export function Copyable({
+  value,
+  onSuccess,
+  onError,
+  className,
+  children,
+  ...props
+}: PropsWithChildren<CopyableProps>) {
+  const valueFallbackInputRef = useRef(null)
+
+  return (
+    <>
+      <div
```

Comment:
WDYT about changing this to a `button` instead? For better accessibility

Reply:
Affirmative. Done!